WcaOnRails/app/models/competition.rb

```diff
@@ -94,11 +94,11 @@ def info
   before_validation :clone_competition, on: [:create]
   def clone_competition
     if competition_id_to_clone.present?
-      competition_to_clone = Competition.find_by_id(competition_id_to_clone)
+      competition_to_clone = Competition.find_by_slug(competition_id_to_clone)
       if competition_to_clone
         attributes = competition_to_clone.attributes
-        # Don't clone id, name, cellName, and dates.
-        %w(id name cellName year month day endMonth endDay registration_open registration_close results_posted_at).each { |attribute| attributes.delete attribute }
+        # Don't clone id, slug, name, cellName, and dates.
```

Comment:
I fear this comment has started to rot (see `results_posted_at`). How about we remove the comment, or change it to "remove fields we shouldn't clone"?